frontend/src/store/index.js

```diff
@@ -126,6 +126,8 @@ const vendorNameFromImageName = imageName => {
     return 'ubuntu'
   } else if (lowerCaseName.includes('suse') && lowerCaseName.includes('jeos')) {
     return 'suse-jeos'
+  } else if (lowerCaseName.includes('suse') && lowerCaseName.includes('chost')) {
+    return 'suse-chost'
   }
   return undefined
 }
```

Comment:
`vendorNeedsLicense` needs to be adapted to `suse-chost` as well, right?

Reply:
good catch